### DISCUSSION TITLE
Command for readme on repos

### DISCUSSION THREAD

**Discussion by superduppperme** [lerenn/asyncapi-codegen#176] [Q&A]
Everytime I create a new repository, I see these instructions:

![image](https://github.com/nodeschool/discussions/assets/159910556/39430225-a7e5-4e73-ad6f-4b18fceafa21)

Instead  of using `echo "# nvm-extension" >> README.md`, wouldn't it be correct to use `echo "# nvm-extension" > README.md` instead since there is no README.md in this new repository?

**Comment by lerenn** [comment]
Hello @superduppperme !

Unfortunately I think you are on the wrong place, it seems that you are trying to clone one of your repository (not this one) and also that you are asking to change a script that is generated by Github and cannot be changed.

But if I'm wrong, feel free to correct me :)
